packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -22,7 +22,11 @@ interface IgxSchematicContext extends SchematicContext {
 
 export function newProject(options: OptionsSchema): Rule {
 	return (_host: Tree, _hostContext: IgxSchematicContext) => {
-		App.initialize();
+		App.initialize("angular-cli");
+		GoogleAnalytics.post({
+			t: "screenview",
+			cd: "New"
+		});
 		let projLibrary: ProjectLibrary;
```

Comment:
There is another post in the `ig new` logic that will be skipped here: 
When the user chooses a project name, framwork and template, there's a post that [contains information about the choice](https://github.com/IgniteUI/igniteui-cli/blob/vNext/packages/cli/lib/commands/new.ts#L115). This should also be added to the `new` schematic.